quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/config/SolverBuildTimeConfig.java

```diff
@@ -18,37 +17,40 @@
  * See also {@link SolverRuntimeConfig}
  */
 @ConfigGroup
-public class SolverBuildTimeConfig {
+public interface SolverBuildTimeConfig {
```

Comment:
Have you made sure none of the pre-existing behavior have changed? (In other words: is this 100 % backwards compatible?)

Have you ensured that for the single instance, the config is still properly overriden when using the config XML as well as these properties?

Have you ensured the same with the benchmark config?

Reply:
In short, we have integration tests to validate all changes regarding the default/old and new behaviors.

Also, I've used the quickstart project to validate the changes, including the benchmark.